src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -191,4 +194,36 @@ public DescriptorImpl() {
         }
     }
 
+    @Ignore
+    @Issue("JENKINS-30206")
```

Comment:
It is not as this has nothing to do with job properties or parameter definitions.